### PR TITLE
MMCA-3046 sub09 to return consentToDisclosureOfPersonalData

### PR DESCRIPTION
**Pull request by zachhall1234**

### CODE REVIEW

File: app/connectors/Sub09Connector.scala [hunk 1]
```diff
@@ -77,10 +77,9 @@ class Sub09Connector @Inject()(appConfig: AppConfig,
 
     metricsReporter.withResponseTimeLogging("mdg.get.company-information") {
       http.GET[Option[MdgSub09CompanyInformationResponse]](uri, headers = headers)
-        .map(_.map(v => CompanyInformation(v.name, v.address))).recover {
+        .map(_.map(v => CompanyInformation(v.name, v.consent.get, v.address))).recover {
```
Comment (derekspalmer) [inline app/connectors/Sub09Connector.scala]: As the consent field is optional it's better that we do a getOrElse otherwise if it's None we'll get a NoSuchElementException.

File: app/models/CompanyInformation.scala [hunk 2]
```diff
@@ -18,7 +18,7 @@ package models
 
 import play.api.libs.json.{Json, OFormat}
 
-case class CompanyInformation(name: String, address: AddressInformation)
+case class CompanyInformation(name: String, consent: String, address: AddressInformation)
```
Comment (derekspalmer) [inline app/models/CompanyInformation.scala:21]: Should consent not be optional?

File: test/connectors/Sub09ConnectorSpec.scala [hunk 3]
```diff
@@ -100,7 +100,7 @@ class Sub09ConnectorSpec extends SpecBase {
 
     def mdgCompanyInformationResponse(value: JsValue): MdgSub09CompanyInformationResponse = MdgSub09CompanyInformationResponse.sub09CompanyInformation.reads(value).get
 
-    val companyInformation: CompanyInformation = CompanyInformation("Example Ltd", AddressInformation("Example Rd", "Example", Some("AA00 0AA"), "GB"))
+    val companyInformation: CompanyInformation = CompanyInformation("Example Ltd", "1", AddressInformation("Example Rd", "Example", Some("AA00 0AA"), "GB"))
```
Comment (derekspalmer) [inline test/connectors/Sub09ConnectorSpec.scala:112]: Can we add a test for consent being None?

**Review by derekspalmer — APPROVED** [review]

File: app/models/CompanyInformation.scala (hunk 2 above)
Comment (zachhall1234) [reply app/models/CompanyInformation.scala:21]: change made in customs data store to default consent none to "0" for false.
